### DISCUSSION TITLE
4.0.0.Alpha3 failed my tests with WildFly 26.0.1.Final

### DISCUSSION THREAD

**Discussion by hantsy** [wildfly/wildfly-arquillian#206] [Q&A] [answered]
In my Jakarta EE 8 project, the WildFly Embedded Adapter failed now, see:

See: https://github.com/hantsy/jakartaee8-starter-boilerplate/runs/6445144354?check_suite_focus=true#step:5:15660

**Comment by jamezp** [comment]
It sees to pass for me locally. One note is if you're going to set the Log Manager to the JBoss Log Manager you should add it to your test class path in profile. The other option is to remove the `<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>` system property from the `maven-failsafe-plugin`.

**Reply by hantsy** [reply]
Remove the logging option, but it's still failed on Github actions.

**Comment by hantsy** [comment, marked as answer]
When going through the logging details, and use a JDK 11 to run it, it works now.

**Reply by jamezp** [reply]
This also passed on Java 8 for me locally too:
```
openjdk version "1.8.0_332"
OpenJDK Runtime Environment (build 1.8.0_332-b09)
OpenJDK 64-Bit Server VM (build 25.332-b09, mixed mode)
```

**Reply by hantsy** [reply]
Really? I dig into the logs, found an error like *can not load module java.transaction.xa*.

**Reply by hantsy** [reply]
I am using a zulu JDK in the Github actions, see: https://github.com/hantsy/jakartaee8-starter-boilerplate/blob/master/.github/workflows/it-with-arq-wildfly-embedded.yml
, when updating to JDK version 11, it works again.
